src/parsec_client/operations/list_keys.md

```diff
@@ -0,0 +1,37 @@
+# ListKeys
+
+Lists all keys belonging to the application in all providers. Opcode: 26 (`0x001A`)
```

Comment:
Would it make sense to have this work per-provider, and if it's sent to the `CoreProvider` it returns all keys from all crypto providers?

Reply:
That would be a nice addition I think. Would we still keep the same `KeyInfo` structure? Because the `id` field (I have to rename it `provider_id`) would be the same then

Reply:
Yeah, why not, clients can just ignore that data if they don't want it.

Reply:
Cool, done!